DESCRIPTION

```diff
@@ -1,7 +1,7 @@
 Package: CancerEvolutionVisualization
 Title: Publication Quality Phylogenetic Tree Plots
-Version: 3.0.0
-Date: 2024-05-07
+Version: 2.1.0
```

Comment:
why are we reverting back to 2.1?

Reply:
Good catch. I think Git was confused about which change took priority. I'll change it to 3.0.0.